node_modules/col-activities/tests/test-daily.js

```diff
@@ -0,0 +1,788 @@
+
+      // Assert a full link to the asset is present
+      var expectedLink = getAssetLibraryBaseUrl(email.course) + '?col_asset=' + asset.id;
+      assert.ok(email.email.html.indexOf(expectedLink) > -1);
+    });
+  };
+
+  /**
+   * Assert that an email contains references to (and only to) a set of users
+   *
+   * @param  {Object[]}   email     Collected email (as per ActivitiesTestUtil.assertDailyNotifications)
+   * @param  {User[]}     users    The set of users that should be in the email
+   * @api private
+   */
+  var assertEmailContainsUsers = function(email, users) {
+    // Get all the user ids in the email
+    var re = new RegExp(getAssetLibraryBaseUrl(email.course) + '\\?col_user=(\d+)');
+    var userIds = getIdsFromURLs(email, re);
+
+    // Ensure each user in the email is supposed to be referenced
+    _.each(userIds, function(id) {
+      assert.ok(_.find(users, {'id': id}));
+    });
+
+    // Ensure all users are present both by full name and URL
+    _.each(users, function(user) {
+      // Assert the name of the user is present in the email
+      assert.ok(email.email.html.indexOf(user.canvas_full_name) > -1);
+
+      // Assert a full link to the user's asset library is present
+      var expectedLink = getAssetLibraryBaseUrl(email.course) + '?col_user=' + user.id;
+      assert.ok(email.email.html.indexOf(expectedLink) > -1);
+    });
+  };
+
+  /**
+   * Assert that an email contains references to (and only to) a set of whiteboards
+   *
+   * @param  {Object[]}         email           Collected email (as per ActivitiesTestUtil.assertDailyNotifications)
+   * @param  {Whiteboard[]}     whiteboards     The set of whiteboards that should be in the email
+   * @api private
+   */
+  var assertEmailContainsWhiteboards = function(email, whiteboards) {
+    // Get all the whiteboard ids in the email
+    var re = new RegExp(getCourseUrl(email.course) + '/whiteboards?col_whiteboard=(\d+)');
+    var whiteboardIds = getIdsFromURLs(email, re);
+
+    // Ensure each user in the email is supposed to be referenced
+    _.each(whiteboardIds, function(id) {
+      assert.ok(_.find(whiteboards, {'id': id}));
+    });
+
+    // Ensure all whiteboards are present both by title and URL
+    _.each(whiteboards, function(whiteboard) {
+      // Assert the title of the whiteboard is present in the email
+      assert.ok(email.email.html.indexOf(whiteboard.title) > -1);
+
+      // Assert a full link to the whiteboard is present
+      var expectedLink = getCourseUrl(email.course) + '/whiteboards?col_whiteboard=' + whiteboard.id;
+      assert.ok(email.email.html.indexOf(expectedLink) > -1);
+    });
+  };
+
+  /**
+   * Get the tool URL for the asset library
+   *
+   * @param  {Course} course The course for which to get the asset library URL
+   * @return {String}        The URL towards the asset library
+   * @api private
+   */
+  var getAssetLibraryBaseUrl = function(course) {
+    return getCourseUrl(course) + '/assetlibrary';
```

Comment:
Isn't a tool's URL always `/external_tools/<toolid>` instead of `/external_tools/assetlibrary` ?